templates/organization/organization_list.html

```diff
@@ -263,44 +264,57 @@ <h1>Endorsers</h1>
 {#    </p>#}
 
     <script>
+        function displayLoadingBanner() {
+            document.getElementById('FilterPopupMessage').style.display = 'block';
+        }
         $(function() {
             $('#limit_to_opinions_in_this_year_id').change(function() {
+                displayLoadingBanner();
                 this.form.submit();
             });
             $('#limit_to_opinions_in_state_code_id').change(function() {
+                displayLoadingBanner();
                 this.form.submit();
             });
             $('#organization_type_filter_id').change(function() {
+                displayLoadingBanner();
                 this.form.submit();
             });
             $('#show_all_id').change(function() {
+                 displayLoadingBanner();
```

Comment:
Hi @itcreativeusa I am going to approve this pull request, but when you get a chance please make sure the indenting of this line (and the other "displayLoadingBanner" lines further down in this file) matches the lines immediately after it. This line is indented 1 space more than the lines of code immediate after it.